ooni/utils/net.py

```diff
@@ -27,8 +27,6 @@ def buildProtocol(self, addr):
 # agents with largest anonymity set.
 
 userAgents = ("Mozilla/5.0 (Windows; U; Windows NT 5.1; en-US; rv:1.9.1.7) Gecko/20091221 Firefox/3.5.7",
-              "Mozilla/5.0 (iPhone; U; CPU iPhone OS 3 1 2 like Mac OS X; en-us)"
-              "AppleWebKit/528.18 (KHTML, like Gecko) Mobile/7D11",
```

Comment:
Also this change was already discussed in #395. I understand that picking a browser identification at random could be a problem when you pick a mobile browser in one run and a non-mobile browser in the control run, since that leads to the mobile site being different from the desktop one to be identified as censorship. @hellais, @anadahz do you have any suggestion on how to deal with this issue?